_contentTemplates/common/get-started.md

```diff
@@ -175,8 +174,7 @@ To setup a local NuGet package source, so you can install the Telerik components
         </TelerikRootComponent>
         
 
-1. Open the main layout file (by default, the `~/Shared/MainLayout.razor` file in the Blazor project), and add `@layout TelerikLayout` as the *first line* in the file:
-
+1. Open the main layout file (by default, the `~/Shared/MainLayout.razor` file in the Blazor project), and add `@layout TelerikLayout` as the *first line* in the file. This will ensure that the `TelerikRootComponent` wraps all the content in `MainLayout`. Alternatively, the `TelerikRootComponent` can reside directly in `MainLayout`, but we place it in another file for better separation of concerns.
```

Comment:
Not sure if this addition will make it clearer, or will add confusion. Early versions had similar long-winded explanations that gave people options, and that caused questions and concerns like "which is better" or "which should I do".

Reply:
The current docs also triggers questions, which is the reason for the update. I added what we recommend at both places, so novice devs should have a clear choice, while the rest should be savvy enough.